ts/utils/checks.ts

```diff
@@ -0,0 +1,22 @@
+import { Wallet } from "../types/pagopa";
```

Comment:
Rename this file to "wallet.ts" as suggested.

Reply:
ok